### PR TITLE
set data-analysis on sample in status and LIMS

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**
Set data analysis on sample both in status and LIMS

Contact: @moahaegglund 

**How to test**:
1. install on stage of the hasta: `bash servers/resources/hasta.scilifelab.se/update-cg-stage.sh set-data_analysis-on-sample`
1. activate stage: `us`
1. run following command: `cg set sample LIMSID --data-analysis "custom" SIGN`

**Expected outcome**:
The dataanalysis value should have been set both in status db and LIMs. comment should say so in statusdb.
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by @ingkebil 
- [x] tests executed by @patrikgrenfeldt  
- [x] "Merge and deploy" approved by @patrikgrenfeldt 
Thanks for filling in who performed the code review and the test!

This is minor **version bump** because it adds some func in backwards compatible manner

### CODE REVIEW

File: cg/cli/set.py [hunk 1]
```diff
@@ -55,14 +54,19 @@ def family(context, action, priority, panels, family_id):
...
+                                                               'put text between quotation marks! '
+                                                               'This will not overwrite the '
+                                                               'current comment.')
 @click.option('-d', '--downsampled-to', type=int, help='sets number of downsampled \
               total reads. Enter 0 to reset.')
 @click.option('-a', '--application-tag', 'apptag', help='sets application tag.')
 @click.option('-k', '--capture-kit', help='sets capture kit.')
+@click.option('--data-analysis', help='sets data-analysis.')
 @click.argument('sample_id')
+@click.argument('user_signature')
```
Comment (ingkebil) [inline cg/cli/set.py]: MAybe not something for now, but we can figure out who is running the command and sign the comment with the logged in user name

File: cg/cli/set.py [hunk 2]
```diff
@@ -145,10 +149,23 @@ def sample(context, sex, customer, comment, downsampled_to, apptag, capture_kit,
 
     if capture_kit:
         sample_obj.capture_kit = capture_kit
-        click.echo(click.style(f"Capture kit {capture_kit} added to sample "
+        click.echo(click.style(f"Capture kit {capture_kit} set on sample "
                                f"{sample_obj.internal_id}", fg='green'))
         context.obj['status'].commit()
 
+    if data_analysis:
+        sample_obj.comment = sample_obj.comment + f"Data-analysis changed from" \
```
Comment (ingkebil) [inline cg/cli/set.py]: This will concat the comments without a separator. Can we add an '\r' or a space?

File: cg/cli/set.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/cli/set.py]: That would be nice, do you know of an example where we do it like that?

File: cg/cli/set.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply cg/cli/set.py]: I'd like to use join here but  maybe later

File: cg/cli/set.py (hunk 1 above)
Comment (ingkebil) [reply cg/cli/set.py]: This should work

```
import getpass
username = getpass.getuser()
```

File: cg/cli/set.py (hunk 2 above)
Comment (patrikgrenfeldt) [reply cg/cli/set.py]: I think my answer was a besides the point, I'll use a separator :)

**Comment by ingkebil** [comment]
Tests are failing horribly. Can you have a look?

**Comment by patrikgrenfeldt** [comment]
image
👍

File: cg/cli/set.py (hunk 1 above)
Comment (patrikgrenfeldt) [reply cg/cli/set.py]: implemented

**Comment by patrikgrenfeldt** [comment]
image
image
👍

**Comment by patrikgrenfeldt** [comment]
deployed

**Pull request merged**